website/deployment.yaml

```diff
@@ -110,3 +110,18 @@ spec:
   selector:
     matchLabels:
       app: website
+---
```

Comment:
I'm not sure where we are on the router elimination, but shouldn't we be modifying router's deployment.yaml?

Reply:
This is all a bit annoyingly interweaved. Right now router's `deployment.yaml` declares the `website` service, and routes traffic to `www` to `https://website/`. If this PR goes in now, this `www` service is entirely unused in default, but gateway will be able to pick it up for `internal.hail.is/<namespace>/www`. When I delete router, I just won't copy over the `website` service from router's deployment.yaml to this one. Does that make sense?